elks/arch/i86/defconfig

```diff
@@ -18,7 +18,7 @@
 # System
 #
 CONFIG_ARCH_IBMPC=y
-# CONFIG_ARCH_8081X is not set
+# CONFIG_ARCH_8018X is not set
```

Comment:
Nice catch!!

Reply:
I did a quick grep / sed to compare the list of CONFIG_* variables documented vs used in the code, curious if there were typos that caused help text to not appear.  There were a couple (this and the compaq one).